src/main/java/io/orangebeard/client/OrangebeardClient.java

```diff
@@ -56,6 +59,10 @@ public UUID startSuite(StartTestItem testItem) {
         return null;
     }
 
+    public UUID startSuite(UUID suiteId, StartTestItem testItem) {
```

Comment:
Ik denk dat het beter is als we startTest startTestItem gaan noemen, en die voor alles gaan aanroepen.